gwctl/pkg/resources/policies/policies.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policies
+
+import (
+	_ "embed"
+	"fmt"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"sigs.k8s.io/yaml"
+
+	"sigs.k8s.io/gateway-api/gwctl/pkg/policymanager"
+	"sigs.k8s.io/gateway-api/gwctl/pkg/types"
+)
+
+func Print(params *types.Params, policies []policymanager.Policy) {
+	sort.Slice(policies, func(i, j int) bool {
+		a := fmt.Sprintf("%v/%v", policies[i].Unstructured().GetNamespace(), policies[i].Unstructured().GetName())
+		b := fmt.Sprintf("%v/%v", policies[j].Unstructured().GetNamespace(), policies[j].Unstructured().GetName())
+		return a < b
+	})
+
+	tw := tabwriter.NewWriter(params.Out, 0, 0, 2, ' ', 0)
+	row := []string{"POLICYNAME", "POLICYKIND", "TARGETNAME", "TARGETKIND"}
+	tw.Write([]byte(strings.Join(row, "\t") + "\n"))
+
+	for _, policy := range policies {
+		row := []string{
+			policy.Unstructured().GetName(),
+			policy.Unstructured().GroupVersionKind().Kind,
+			policy.TargetRef().Name,
+			policy.TargetRef().Kind,
+		}
+		tw.Write([]byte(strings.Join(row, "\t") + "\n"))
+	}
+	tw.Flush()
+}
+
+func PrintCRDs(params *types.Params, policyCRDs []policymanager.PolicyCRD) {
+	sort.Slice(policyCRDs, func(i, j int) bool {
+		a := fmt.Sprintf("%v/%v", policyCRDs[i].CRD().GetNamespace(), policyCRDs[i].CRD().GetName())
+		b := fmt.Sprintf("%v/%v", policyCRDs[j].CRD().GetNamespace(), policyCRDs[j].CRD().GetName())
+		return a < b
+	})
+
+	tw := tabwriter.NewWriter(params.Out, 0, 0, 2, ' ', 0)
+	row := []string{"CRD_NAME", "CRD_GROUP", "CRD_KIND", "CRD_INHERITED", "CRD_SCOPE"}
+	tw.Write([]byte(strings.Join(row, "\t") + "\n"))
+
+	for _, policyCRD := range policyCRDs {
+		row := []string{
+			policyCRD.CRD().Name,
+			policyCRD.CRD().Spec.Group,
+			policyCRD.CRD().Spec.Names.Kind,
+			fmt.Sprintf("%v", policyCRD.IsInherited()),
```

Comment:
Would recommend using a string value here of "Inherited" or "Direct" instead

Reply:
Great idea. Changed.